libs/shared/styles/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './lib/shared-styles';
```

Comment:
Could the tailwind.js config also be included in this shared library, and be added to `payments-next`? `packages/fxa-react/configs/tailwind.js`

This file contains all the shared themes used in the FxA monorepo